### PR TITLE
CHANGE styling of sourcejs.com home-page

### PR DESCRIPTION
**Pull request by ndelangen**
UPGRADE to sourcejs 0.6.0 nightly 5 as a NPM package. This seems to just work™, at least locally.

I also tweaked the visual style and order of things on the home page. I hope you like it.

### CODE REVIEW

**Comment by ndelangen** [comment]
Decided deploying it to heroku would be the best test if it actually works not just on my machine. Now it succesfully deploys to heroku: https://still-beyond-68515.herokuapp.com/

**Comment by robhrt7** [comment]
Hi,

Thanks for the neat design changes you've added. I had some plans on redesign around 0.7.0, but it's quite far away, considering the load on other projects I'm involved right now. These improvements make me less worry about the current design state of sourcejs.com.

Regarding bumping the version, I prefer keeping official website on a stable version, which is 0.5.6 right now, so let's revert this part.

I also have a heroku deployed version - https://sourcejs-nightly.herokuapp.com/, will need to automate that at some point, so latest dev version always deployed to nightly.

File: package.json [hunk 1]
```diff
@@ -2,11 +2,17 @@
   "author": "SourceJS",
   "version": "0.1.0",
   "description": "Sourcejs.com project website",
+	"scripts": {
```
Comment (robhrt7) [inline package.json]: please fix formatting - tabs -> 2 spaces

File: package.json [hunk 2]
```diff
@@ -2,11 +2,17 @@
...
+		"build-local": "grunt default",
+		"build-source": "cd ./node_modules/sourcejs && npm run build",
+    "start": "npm run build-local && npm run build-source && node ./node_modules/sourcejs/app.js"
```
Comment (robhrt7) [inline package.json]: before moving sourcejs to NPM install flow, let's wait before 0.6.0 release. Please revert lines 7-8 and 15.

**Comment by robhrt7** [comment]
Could you also please use the same gitter badge as in the official readme? [Gitter chat](https://gitter.im/sourcejs/Source)

**Comment by ndelangen** [comment]
I'll probably have time to update it tonight.

**Comment by custa1200** [comment]
I have been using the 0.6.0 nightly 5 and except for my JS issues I have mentioned in otherrecent issues it seems to be working fine.

**Comment by robhrt7** [comment]
Great job!

**Pull request merged**